### ISSUE TITLE
set lr_mult, wd_mult of fc7_weight, but it didn't work.

### ISSUE THREAD

**Issue by lly8752** [deepinsight/insightface#830]
_weight = mx.symbol.Variable("fc7_weight", shape=(config.num_classes, config.emb_size), 
        lr_mult=config.fc7_lr_mult, wd_mult=config.fc7_wd_mult, init=mx.init.Normal(0.01))
i set config.fc7_lr_mult=0, config.fc7_wd_mult=0,why _weight update when i train the network

**Comment by latstars** [comment]
I meet the same problem with you @lly8752, the fc7_lr_mult and fc7_wd_mult is no use. Have you figured it out? Would you like to share the solutions? Thanks!

**Comment by latstars** [comment]
I have met your solution in another place ( https://github.com/apache/incubator-mxnet/issues/15718#issuecomment-520278976 ), thanks very much.
The solution is simple, we only need to change the optimizer initialization to "opt = optimizer.SGD(sym=sym, learning_rate=args.lr, momentum=args.mom, wd=args.wd, rescale_grad=_rescale)"

**Comment by latstars** [comment]
> Hi Sirs, is there anyone would like to talk about what’s going on here, why are you guys interested in “f7” things?

The f7 layer is the classifier layer in the face model. I just want to assign a special weight decay and a learning rate to the weight of this layer,  different from the backbone layer.

**Comment by Light--** [comment]
@lly8752 @latstars 
may i ask how do you know the lr_mult didn't work? since we can not output the lr one specific layer...
